tracing/tracing-test.sh

```diff
@@ -23,17 +32,48 @@ jaeger_server=${jaeger_server:-localhost}
 jaeger_ui_port=${jaeger_ui_port:-16686}
 jaeger_docker_container_name="jaeger"
 
+logdir=""
+
 # Cleanup will remove Jaeger container and
 # disable tracing.
-cleanup(){
-	stop_jaeger 2>/dev/null || true
-	"${SCRIPT_PATH}/../.ci/configure_tracing_for_kata.sh" disable
-}
+cleanup()
+{
+	local fp="info"
+	local result="failed"
+	local dest="$logdir"
+
+	if [ "$success" = true ]; then
+		result="passed"
+
+		[ -z "$DEBUG_KEEP_JAEGER" ] && stop_jaeger 2>/dev/null || true
```

Comment:
In case of not success it is not going to stop the Jaeger server. Is it intentional?

Reply:
Yes - with the CI, we don't need to stop the server as the VM env will be destroyed. But if the tests failed, you want to be able to view the Jaeger UI to see why, so the server needs to be running.